docs/source/runtime-python-api-reference.rst

```diff
@@ -0,0 +1,17 @@
+ExecuTorch Runtime Python API Reference
+----------------------------------
+The Python ``executorch.runtime`` module wraps the C++ ExecuTorch runtime. It can load and execute serialized ``.pte`` program files: see the `Export to ExecuTorch Tutorial <tutorials/export-to-executorch-tutorial.html>`__ for how to convert a PyTorch ``nn.Module`` to an ExecuTorch ``.pte`` program file. Execution accepts and returns ``torch.Tensor`` values, making it a quick way to validate the correctness of the program.
```

Comment:
Looking at the rendered web page, I see

> The Python executorch.runtime module wraps the C++ ExecuTorch runtime. It can load and execute serialized .pte program files: see the [Export to ExecuTorch Tutorial](https://docs-preview.pytorch.org/pytorch/executorch/6276/tutorials/export-to-executorch-tutorial.html) for how to convert a PyTorch nn.Module to an ExecuTorch .pte program file. Execution accepts and returns torch.Tensor values, making it a quick way to validate the correctness of the program. For detailed information on how APIs evolve and the deprecation process, please refer to the [ExecuTorch API Life Cycle and Deprecation Policy](https://docs-preview.pytorch.org/pytorch/executorch/6276/api-life-cycle.html).
>
> Interface to the native C++ ExecuTorch runtime.

Since the .rst text and this doc comment flow together, it might make sense to move more of the details into the python file's doc comment. Or, add a section like `## executorch.runtime Module` in this file before the `automodule::` to make this separation a little more natural. Right now it's a little jarring.